strings.po

```diff
@@ -4817,7 +4817,7 @@ msgstr "Build"
 #. STRINGS.DUPLICANTS.CHORES.BUILD.STATUS
 msgctxt "STRINGS.DUPLICANTS.CHORES.BUILD.STATUS"
 msgid "Going to build"
-msgstr "Baut grade"
+msgstr "Baut gerade"
```

Comment:
Die ganzen "Going to XY", sind das wirklich "Tut grade XY"-Texte oder nicht eher Pläne, also im Sinne von "Wird gleich XY tun"? Müsste man im Spiel mal testen, wann das auftaucht.